### PR TITLE
Perform CI over Erlang 24

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
CI duration went down from 1m 53s to 35s using cache. 🎉

File: rebar.config [hunk 1]
```diff
@@ -65,3 +65,14 @@
...
+{hex, [
+    {doc, #{provider => edoc}},
+    {repos, [
+        #{ name => <<"hexpm">>,
+           api_url => <<"http://localhost:4000/api">>,
+           repo_url => <<"http://localhost:4000">>,
+           repo_verify => false,
+           repo_verify_origin => false}
+    ]}
+]}.
```
Comment (g-andrade) [inline rebar.config]: ^ What is the meaning of this?

Comment (paulo-ferraz-oliveira) [reply rebar.config]: It was an unfortunate push... **edit**: ... now fixed.

Comment (g-andrade) [reply rebar.config]: 👍 at least I had the chance of dramatically asking "What is the meaning of this?"

**Review by g-andrade — APPROVED** [review]

**Pull request merged**